server/sonar-server/src/main/java/org/sonar/server/serverid/ws/GenerateAction.java

```diff
@@ -79,13 +78,10 @@ public void define(WebService.NewController controller) {
 
   @Override
   public void handle(Request request, Response response) throws Exception {
-    userSession.checkPermission(SYSTEM_ADMIN);
+    userSession.checkLoggedIn().checkIsRoot();
```

Comment:
description should be fixed, root is now required, not system administration... or is that the name we give to root?